src/test/ui/weird-exprs.rs

```diff
@@ -7,6 +7,8 @@
 #![allow(dead_code)]
 #![allow(unreachable_code)]
 #![allow(unused_braces, unused_must_use, unused_parens)]
```

Comment:
The `allow` could be local to `confusable_non_ascii`, not global to the module, so it doesn't inadvertently affect other code and the point of `confusable_non_ascii` is more clear.

Reply:
> `warning: allow(confusable_idents) is ignored unless specified at crate level`

Reply:
😒